### ISSUE TITLE
PS1 LUCIFERD - "VAB" Instrument Set conversion doesn't make a .SF2 or .DLS file

### ISSUE THREAD

**Issue by upside666table** [vgmtrans/vgmtrans#163]
I'm trying to extract the music from this PS1 NTSC-J game called "Luciferd - Psychological Adventure [SLPS-00811]". 

The music files are contained in the file "BGM.ARC". VGMTrans can detect SEQ sequence files and VAB instrument sets cointaned in the file. It sucessfully converts the SEQ sequence files, 
but fails to convert the VAB files into sf2 or dls (with 1 exception out of 16).
The log lists 4 warnings:

Too many tones (112) in Program # 126.
Too many tones (112) in Program # 126.
Too many tones (192) in Program # 127. 
Too many tones (201) in Program # 127.

VGMTrans version: VGMTrans Interim Build (2017-02-25).

**Comment by sykhro** [comment]
I have no way of obtaining the original files. I can look into this if you contact me privately [here](mailto:eli@sykh.ro)

**Comment by Meerkov** [comment]
Sounds like this https://github.com/vgmtrans/vgmtrans/issues/251

**Comment by Meerkov** [comment]
Indeed, please follow the instructions I posted in #251. I was able to get this working with no issues.

**Comment by mikelow** [comment]
This is resolved.
